tasks-by-users/AbbasShaikh0304/01-E-Commerce website/Style.css

```diff
@@ -0,0 +1,601 @@
+  margin-bottom: 50px;
+  padding: 10px;
+
+  transition: transform 0.5s;
+
+  cursor: pointer;
+}
+.col-4:hover {
+  transform: translateY(-5px);
+}
+.col-4 img {
+  width: 100%;
+}
+.title {
+  position: relative;
+  text-align: center;
+
+  margin: 0 auto 80px;
+  line-height: 60px;
+
+  color: #555;
+}
+.title::after {
+  content: "";
+
+  width: 80px;
+  height: 5px;
+
+  position: absolute;
+  bottom: 0;
+  left: 50%;
+
+  background: #ff523b;
+  border-radius: 5px;
+
+  transform: translateX(-50%);
+}
+h4 {
+  color: #555;
+
+  font-weight: normal;
+}
+.col-4 p {
+  font-size: 14px;
+}
+.rating .fa {
+  color: #ff523b;
+}
+
+/* ------------offer-------------*/
+
+.offer {
+  margin-top: 80px;
+  padding: 30px 0;
+
+  background: radial-gradient(#fff, #ffd6d6);
+}
+.col-2 .offer-img {
+  padding: 50px;
+}
+small {
+  color: #555;
+}
+
+/*-----------testimonial------------*/
+
+.testimonial {
+  padding-top: 100px;
+}
+.testimonial .col-3 {
+  padding: 40px 20px;
+  text-align: center;
+
+  box-shadow: 0 0 20px 0px rgba(0, 0, 0, 0.1);
+
+  cursor: pointer;
+
+  transition: transform 0.5s;
+}
+.testimonial .col-3 img {
+  width: 50px;
+
+  margin-top: 20px;
+
+  border-radius: 50%;
+}
+.testimonial .col-3:hover {
+  transform: translateY(-10px);
+}
+.fa.fa-quote-left {
+  font-size: 34px;
+
+  color: #ff523b;
+}
+.col-3 p {
+  font-size: 12px;
+
+  margin: 12px 0;
+
+  color: #777;
+}
+.testimonial .col-3 h3 {
+  color: #555;
+
+  font-size: 16px;
+  font-weight: 600;
+}
+
+/* --------- Brands ----------- */
+
+.brands {
+  margin: 100px auto;
+}
+.col-5 {
+  width: 160px;
+}
+.col-5 img {
+  width: 100%;
+
+  cursor: pointer;
+
+  filter: grayscale(100%);
+}
+.col-5 img:hover {
+  filter: grayscale(0);
+}
+/* ---------- footer -------- */
+footer {
+  padding: 60px 0 20px;
+
+  color: #8a8a8a;
+  background-color: #000;
+
+  font-size: 14px;
+}
+footer p {
+  color: #8a8a8a;
+}
+footer h3 {
+  margin-bottom: 20px;
+
+  color: #fff;
+}
+.footer-col-1,
+.footer-col-2,
+.footer-col-3,
+.footer-col-4 {
+  margin-bottom: 20px;
+
+  min-width: 250px;
+}
+.footer-col-1 {
+  flex-basis: 30%;
+}
+.footer-col-2 {
+  flex: 1;
+  text-align: center;
+}
+.footer-col-2 img {
+  margin-bottom: 20px;
+
+  width: 180px;
+}
+.footer-col-3,
+.footer-col-4 {
+  flex-basis: 12%;
+  text-align: center;
+}
+ul {
+  list-style: none;
+}
+.app-logo {
+  margin-top: 20px;
+
+  text-align: center;
+}
+.app-logo img {
+  width: 150px;
+}
+footer hr {
+  margin: 20px 0;
+
+  height: 1px;
+
+  background-color: #8a8a8a;
+
+  border: none;
+}
+.copyright {
+  text-align: center;
+}
+.menu-icon {
+  display: none;
+
+  width: 28px;
+
+  margin-left: 20px;
+}
+
+/*-----------All Products----------*/
+
+.row-2 {
+  margin: 100px auto 50px;
+
+  justify-content: space-between;
+}
+select {
+  padding: 5px;
+
+  border: 1px solid #ff523b;
+}
+select:focus {
+  outline: none;
+}
+.page-btn {
+  margin: 0 auto 80px;
+}
+.page-btn span {
+  display: inline-block;
+  width: 40px;
```

Comment:
here @AbbasShaikh0304

Reply:
told you about width and height @Tauqeer-Ahmed-99

Reply:
but display and width should not be grouped together right?? @AbbasShaikh0304

Reply:
I didn't aware of this, i've arrange in the manner i thought it will look good. @Tauqeer-Ahmed-99